src/store/actions/init-words.js

```diff
@@ -1,8 +1,12 @@
-import { getWords } from '../../service/get-words';
+import { getRandomIndexInRange } from '../../helpers';
 
 export const INIT_WORDS = 'INIT_WORDS';
 
-export const initWords = () => ({
-    type: INIT_WORDS,
-    words: getWords(),
-});
+export const initWords = (allWords) => {
+    const wordsCount = allWords.length;
+    const index = getRandomIndexInRange(wordsCount);
+    return {
+        type: INIT_WORDS,
+        words: allWords ? [...allWords[index]] : [],
```

Comment:
oh, please ignore! now I remember why it is this way!

I've mentioned it somewhere but it would be cool to have a feature of avoiding the same `words` to be drawn twice. We could store "already played words" in localStorage.